README.md

```diff
@@ -10,7 +10,7 @@ All integration options require a Spreedly account and an environment key. See [
 ## Installation
 We recommend using [Gradle](https://docs.gradle.org/current/userguide/userguide.html) to integrate the Spreedly SDK with your project. the `Spreedly` package provides basic, low-level APIs for custom integrations. The `Express` package provides custom controls and the Spreedly Express workflow, a prebuilt UI for collecting and selecting payment methods.
 
-First add the following to your [repositories section of your build.gradle file](https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:declaring_multiple_repositories):
+First add the following to your [repositories section of your build.gradle file](https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:declaring_multiple_repositories), or [settings.gradle file](https://docs.gradle.org/current/userguide/declaring_repositories.html#sub:centralized-repository-declaration), depending on your preferred setup:
```

Comment:
This is certainly my own ignorance of Android Development, but in the sample app that Android Studio created for me, the latter was true so felt it could be added.

Reply:
this is not done on android. I would remove this addition.